packages/eslint-config-godaddy-flow/package.json

```diff
@@ -24,25 +25,17 @@
   "bin": {
     "eslint-godaddy-flow": "bin/eslint-godaddy-flow"
   },
-  "dependencies": {
-    "eslint-config-godaddy": "^4.0.0"
-  },
   "peerDependencies": {
-    "babel-eslint": ">=7.2.1 <11.0.0",
-    "eslint": "^6.1.0",
-    "eslint-plugin-babel": "^5.3.0",
-    "eslint-plugin-flowtype": "^4.2.0",
-    "eslint-plugin-json": "^1.4.0",
-    "eslint-plugin-mocha": "^5.3.0"
+    "eslint": ">=6.0.0"
+  },
+  "dependencies": {
+    "babel-eslint": "*",
```

Comment:
I love these being regular dependencies but don't love them being `*` versioned. It's not uncommon to encounter breakages due to differing compatibility with different `eslint` versions and in the case of `babel-eslint` babel versions. I'd rather using dependencies tied a semver major range.

Reply:
💯 `^1.2.3` + renovate/greenkeeper/etc would go a long way here.

Reply:
Sounds good... I'll keep eslint as a peer because the other plugins do that anyway...
I'll update the PR now

Reply:
Updated based on the suggestions here.
unrelated: we should also probably create a separate PR removing the es5 package, it's just one more thing to maintain and it's probably not used by anyone.